package.json

```diff
@@ -19,7 +19,7 @@
     "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-jest": "^21.17.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
-    "eslint-plugin-react": "^7.10.0",
+    "eslint-plugin-react": "7.10.0",
```

Comment:
?

Reply:
Ah, there was an obscure issue in CI (if I recall correctly) with a patch release of this module. I suspect it's since been fixed.